src/test/getarg_tests.cpp

```diff
@@ -33,6 +33,8 @@ static void ResetArgs(const std::string& strArg)
     BOOST_CHECK(gArgs.ParseParameters(vecChar.size(), vecChar.data(), error));
 }
 
+ArgsManager argsman;
```

Comment:
In commit "Include an unused ArgsManager" (2e0c74b1ffe9bd1867700fd4a58458160324e23c)

This variable is still shared between test instances (and I think also accesible outside the test file using `extern`). I think it would be better to add a member variable to the boost test setup so the tests are isolated. This would look something like 

https://github.com/bitcoin/bitcoin/blob/db762529d6bffcc6ab0c1c64b9378925f2d75fa1/src/test/getarg_tests.cpp#L16-L23

https://github.com/ryanofsky/bitcoin/commit/db762529d6bffcc6ab0c1c64b9378925f2d75fa1

Reply:
Thank you, will fix!